processor/README.md

```diff
@@ -244,6 +245,37 @@ span:
     separator: <value>
 ```
 
+In the second form it takes a list of regular expressions to match span name against
+and extract attributes from based on subexpressions.
+
+`rules` is a list of rules to extract attribute values from span name. The values
+in the span name are replaced by extracted attribute names. Each rule in the list
+is regex pattern string. Span name is checked against the regex and if the regex
+matches then all named subexpressions of the regex are extracted as attributes
+and are added to the span. Each subexpression name becomes an attribute name and
+subexpression matched portion becomes the attribute value. The matched portion
+in the snap name is replaced by extracted attribute name. If the attributes
+already exist in the span then they will be overwritten. The process is repeated
+for all rules in the order they are specified. Each subsequent rule works on the
+span name that is the output after processing the previous rule.
+
+`break_after_match` specifies if processing of rules should stop after the first
```

Comment:
nit: I expect the default to be `false`, because of zero-value, but it isn't clear from the text.

Reply:
Clarified.